package.json

```diff
@@ -4875,6 +4877,622 @@
             }
           }
         ]
+      },
+      {
+        "type": "monovsdbg",
+        "label": ".NET Core using Mono Runtime",
+        "hiddenWhen": "dotnet.debug.serviceBrokerAvailable",
+        "languages": [
+          "csharp"
+        ],
+        "variables": {
```

Comment:
I don't think any of these variables make sense for mono since mono attached by port.